region/brinstar/pink.json

```diff
@@ -904,15 +927,33 @@
                     {
                       "id": "F",
                       "requires": [
-                        "Super",
-                        {"ammo": {
-                          "type": "Super",
-                          "count": 1
-                        }},
+                        {"ammo":{"type": "Super","count": 1}},
                         "h_canBombThings"
                       ]
                     }
                   ]
+                },
+                {
+                  "name": "Speedball",
+                  "notable": false,
+                  "requires": [
+                    "canMockball",
+                    "canTrickyJump",
+                    "canSlowShortCharge",
+                    {"canComeInCharged": {
+                      "framesRemaining": 180,
+                      "fromNode": 7,
+                      "shinesparkFrames": 0,
+                      "unusedTiles": 2
+                    }}
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "F",
+                      "requires": [{"ammo":{"type": "Super","count": 1}}]
+                    }
+                  ],
+                  "devNote": "FIXME: There are fancy ways to jump into the room and shoot the super block and mockball at full speed."
```

Comment:
The fancy strats would be notable for being extremely precise.

This could use a note mentioning the super shot.  Hitting the block doesn't seem notable at the expected skill level.